go.mod

```diff
@@ -3,7 +3,8 @@ module github.com/celestiaorg/celestia-node
 go 1.16
 
 require (
-	github.com/celestiaorg/celestia-core v0.0.1-mvp-das-lightclient.0.20210831102741-321cdc4f8dc4
+	github.com/celestiaorg/celestia-core v0.0.1-mvp-das-lightclient.0.20210922162040-a9b7b4818e8d
```

Comment:
This will soon have to change back to `github.com/tendermint/tendermint` with a replace directive below that points to the fork. Evan can surely help with that though.

Reply:
Nice!